src/routers/alpha-router/gas-models/tick-based-heuristic-gas-model.ts

```diff
@@ -183,18 +183,25 @@ export abstract class TickBasedHeuristicGasModelFactory<
         ) as CurrencyAmount;
 
         // Convert gasCostInTermsOfAmountToken to quote token using execution price
-        const syntheticGasCostInTermsOfQuoteToken = executionPrice.quote(
-          gasCostInTermsOfAmountToken
-        );
+        let syntheticGasCostInTermsOfQuoteToken: CurrencyAmount | null;
+        try {
+          syntheticGasCostInTermsOfQuoteToken = executionPrice.quote(
+            gasCostInTermsOfAmountToken
+          );
+        } catch (err) {
```

Comment:
nit: we probably want to inspect the error log to be sure it's division by zero to swallow the error. I'm erring on the conservative side, in terms of slightly changing the behavior.